logstash-core/src/main/java/org/logstash/ext/JrubyAckedReadClientExt.java

```diff
@@ -27,8 +27,8 @@
     private final ConcurrentHashMap<Long, Long> inflightClocks = new ConcurrentHashMap<>();
 
     private IRubyObject queue;
-    private IRubyObject batchSize;
-    private IRubyObject waitForMillis;
+    private IRubyObject batchSize = RubyNumeric.int2fix(RubyUtil.RUBY, 125);
+    private IRubyObject waitForMillis = RubyNumeric.int2fix(RubyUtil.RUBY, 50);
```

Comment:
These default values were necessary for the rspec tests since they do not use `java_pipeline` where these values are always explicitly set via the `set_batch_dimensions` method.